test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Metadata/DefaultModelMetadataProviderTest.cs

```diff
@@ -45,6 +45,24 @@ public void GetMetadataForType_Cached()
         }
 
         [Fact]
+        public void GetMetadataForObjectType_Cached()
+        {
+            // Arrange
+            var provider = CreateProvider();
+            
+            // Act
+            var metadata1 = Assert.IsType<DefaultModelMetadata>(provider.GetMetadataForType(typeof(object)));
+            var metadata2 = Assert.IsType<DefaultModelMetadata>(provider.GetMetadataForType(typeof(object)));
+
+            // Assert
+            Assert.Same(metadata1, metadata2);
+            Assert.Same(metadata1.Attributes, metadata2.Attributes);
+            Assert.Same(metadata1.BindingMetadata, metadata2.BindingMetadata);
+            Assert.Same(metadata1.DisplayMetadata, metadata2.DisplayMetadata);
+            Assert.Same(metadata1.ValidationMetadata, metadata2.ValidationMetadata);
```

Comment:
Don't bother with the additional assertions. Nothing can be different between properties of `metadata1` and `metadata2` once you've confirmed they are the same.

In addition, the `Assert.IsType()` checks definitely don't belong w/in `// Act`. Why are they needed at all?